packages/mobile/src/notifications.ts

```diff
@@ -48,7 +48,7 @@ class PushNotifications {
         body
       })
     )
-    this.navigation?.navigate(payload)
+    this.navigation?.navigate(payload?.data?.data ?? payload?.data ?? payload)
```

Comment:
is data.data because of axios?

Reply:
honestly not sure, this was a check from previous impl, so it might just be wrong. im adding both just to make sure it actually works before then figuring out which one works :)

Reply:
its really hard to test this locally unfortunately